StreamChat.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -19,6 +19,15 @@
           "version": "4.0.4"
         }
       },
+      {
+        "package": "SnapshotTesting",
+        "repositoryURL": "https://github.com/pointfreeco/swift-snapshot-testing.git",
+        "state": {
+          "branch": null,
+          "revision": "c466812aa2e22898f27557e2e780d3aad7a27203",
+          "version": "1.8.2"
+        }
+      },
```

Comment:
It's not clear from the diff but did you add the target just to the test target?

Reply:
Yes, I've added the snapshot framework only to the test target. But I was expecting this to show in the diff? 🤔 Maybe in package.swift? But I can see that we are not specifying there the dependencies of the test targets, so maybe it is the project file, but it is not clear in the diff?